tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -145,8 +200,75 @@ def test_positive_validate_katello_certs_check_output(self, cert_setup, cert_dat
             )
         self.validate_output(result, cert_data)
 
+    @pytest.mark.tier1
+    def test_katello_certs_check_output_wildcard_inputs(self, generate_certs, cert_data):
+        """Validate that katello-certs-check generates correct output with wildcard certs.
+
+        :id: 7f9da806-5b23-11eb-b7ea-d46d6dd3b5b2
+
+        :steps:
+
+            1. Generate the valid wildcard certs using shell script
```

Comment:
As this test does not generate the certs, would it not be better to say something like:
1. Get valid wildcard certs from `generate_certs`
or:
1. Call `generate_certs` to get valid wildcard certs